src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -541,13 +526,13 @@ public async Task Pound_i_nuget_with_multi_submissions_combines_the_Text_Updates
 
             using var events = result.KernelEvents.ToSubscribedList();
 
-            // For the DisplayedValueUpdated events strip out the restore sources
-            // Verify that they match the expected values
-            events.OfType<DisplayedValueUpdated>()
-                  .Last().FormattedValues
+            events.OfType<DisplayedValueProduced>()
                   .Should()
-                  .ContainSingle(e => e.MimeType == HtmlFormatter.MimeType)
+                  .ContainSingle(v => v.Value is InstallPackagesMessage)
                   .Which.Value
+                  .As<InstallPackagesMessage>()
+                  .RestoreSources
+                  .Aggregate((s, acc) => acc + " & " + s)
```

Comment:
Instead of `Aggregate` why not use `BeEquivalentTo`? It's stricter and simpler.

Reply:
done